tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalServerDelegate.java

```diff
@@ -51,22 +51,7 @@ public PortalServerDelegate()
     @Override
     public IStatus canModifyModules( IModule[] add, IModule[] remove )
     {
-        IStatus retval = Status.OK_STATUS;
-
-        if( !CoreUtil.isNullOrEmpty( add ) )
-        {
-            for( IModule module : add )
-            {
-                if( !"liferay.bundle".equals( module.getModuleType().getId() ) )
-                {
-                    retval =
-                        LiferayServerCore.error( "Unable to add module with type " + module.getModuleType().getName() );
-                    break;
-                }
-            }
-        }
-
-        return retval;
+        return Status.OK_STATUS;
```

Comment:
We need to check something here, to make sure that it is at least the list of modules that we support., liferay.bundle plugin the other ones that you just added.